helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -75,9 +77,15 @@ internal class ListLicensesCommand : CliktCommand(
 
     private val onlyOffending by option(
         "--only-offending",
-        help = "Only list licenses causing a rule violation of error severity in the given ORT result."
+        help = "Only list licenses causing a rule violation of severity specified severity, see --severity."
     ).flag()
 
+    private val severity by option(
+        "--severity",
```

Comment:
Just an idea: Given that specifying this option only makes sense if `--only-offending` is also specified, how about making this more obvious by using a common prefix and naming this option `--offending-severity` and rename `--only-offending` to `--offending-only`?

Reply:
Good idea. I'll file another PR for it after this is merged.

Reply:
@sschuberth here you go: #2585.